openapi.go

```diff
@@ -209,6 +214,18 @@ func RegisterOpenAPIOperation[T, B any](openapi *OpenAPI, route Route[T, B]) (*o
 		route.Operation = openapi3.NewOperation()
 	}
 
+	if route.FullName == "" {
+		route.FullName = route.Path
+	}
+
+	if route.Operation.Summary == "" {
+		route.Operation.Summary = route.NameFromNamespace(camelToHuman)
+	}
+
+	if route.Operation.OperationID == "" {
+		route.Operation.OperationID = route.Method + "_" + strings.ReplaceAll(strings.ReplaceAll(route.Path, "{", ":"), "}", "")
```

Comment:
This should be moved to a method, here is looks like an hack to clean things, while it's very important

Reply:
Done. Thanks!

Reply:
Done, thanks!